2025-01-19 12:50:44,700 INFO    Thread-4  :13698 [wandb_setup.py:_flush():71] setting env: {'project': 'Ablation_LLMs', 'entity': 'maestria-julian', 'root_dir': '/home/julian/CATALOG', 'sweep_id': 'md4vsxd6', 'run_id': '61fdt6yv', 'sweep_param_path': '/home/julian/CATALOG/wandb/sweep-md4vsxd6/config-61fdt6yv.yaml'}
2025-01-19 12:50:44,700 INFO    Thread-4  :13698 [wandb_setup.py:_flush():71] setting login settings: {}
2025-01-19 12:50:44,700 INFO    Thread-4  :13698 [wandb_init.py:_log_setup():357] Logging user logs to /home/julian/CATALOG/wandb/run-20250119_125044-61fdt6yv/logs/debug.log
2025-01-19 12:50:44,700 INFO    Thread-4  :13698 [wandb_init.py:_log_setup():358] Logging internal logs to /home/julian/CATALOG/wandb/run-20250119_125044-61fdt6yv/logs/debug-internal.log
2025-01-19 12:50:44,701 INFO    Thread-4  :13698 [wandb_init.py:init():390] calling init triggers
2025-01-19 12:50:44,701 INFO    Thread-4  :13698 [wandb_init.py:init():395] wandb.init called with sweep_config: {'batch_size': 59, 'dropout': 0.3157181561140745, 'hidden_dim': 1006, 'lr': 0.05417961766483669, 'momentum': 0.8174844623712146, 'num_epochs': 161, 'num_layers': 6, 'opt': 'sgd', 'patience': 10, 't': 0.01, 'weight_Clip': 0.6093858303564759}
config: {}
2025-01-19 12:50:44,701 INFO    Thread-4  :13698 [wandb_init.py:init():435] starting backend
2025-01-19 12:50:44,701 INFO    Thread-4  :13698 [backend.py:_multiprocessing_setup():94] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-01-19 12:50:44,712 INFO    Thread-4  :13698 [backend.py:ensure_launched():198] starting backend process...
2025-01-19 12:50:44,720 INFO    Thread-4  :13698 [backend.py:ensure_launched():203] started backend process with pid: 13736
2025-01-19 12:50:44,722 INFO    Thread-4  :13698 [wandb_init.py:init():444] backend started and connected
2025-01-19 12:50:44,726 INFO    Thread-4  :13698 [wandb_run.py:_config_callback():911] config_cb None None {'batch_size': 59, 'dropout': 0.3157181561140745, 'hidden_dim': 1006, 'lr': 0.05417961766483669, 'momentum': 0.8174844623712146, 'num_epochs': 161, 'num_layers': 6, 'opt': 'sgd', 'patience': 10, 't': 0.01, 'weight_Clip': 0.6093858303564759}
2025-01-19 12:50:44,726 INFO    Thread-4  :13698 [wandb_init.py:init():503] updated telemetry
2025-01-19 12:50:44,747 INFO    Thread-4  :13698 [wandb_init.py:init():533] communicating current version
2025-01-19 12:50:45,676 INFO    Thread-4  :13698 [wandb_init.py:init():538] got version response upgrade_message: "wandb version 0.19.4 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2025-01-19 12:50:45,676 INFO    Thread-4  :13698 [wandb_init.py:init():548] communicating run to backend with 30 second timeout
2025-01-19 12:50:46,178 INFO    Thread-4  :13698 [wandb_init.py:init():576] starting run threads in backend
2025-01-19 12:50:48,468 INFO    Thread-4  :13698 [wandb_run.py:_console_start():1693] atexit reg
2025-01-19 12:50:48,469 INFO    Thread-4  :13698 [wandb_run.py:_redirect():1567] redirect: SettingsConsole.REDIRECT
2025-01-19 12:50:48,469 INFO    Thread-4  :13698 [wandb_run.py:_redirect():1572] Redirecting console.
2025-01-19 12:50:48,469 ERROR   Thread-4  :13698 [wandb_run.py:_redirect():1631] Failed to redirect.
Traceback (most recent call last):
  File "/home/julian/miniconda3/envs/CATALOG/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 1624, in _redirect
    out_redir.install()
  File "/home/julian/miniconda3/envs/CATALOG/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 709, in install
    _WSCH.add_fd(self._pipe_read_fd)
  File "/home/julian/miniconda3/envs/CATALOG/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 659, in add_fd
    self._register()
  File "/home/julian/miniconda3/envs/CATALOG/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 644, in _register
    old_handler = signal.signal(signal.SIGWINCH, lambda *_: None)
  File "/home/julian/miniconda3/envs/CATALOG/lib/python3.8/signal.py", line 47, in signal
    handler = _signal.signal(_enum_to_int(signalnum), _enum_to_int(handler))
ValueError: signal only works in main thread
2025-01-19 12:50:48,473 INFO    Thread-4  :13698 [wandb_init.py:init():603] run started, returning control to user process
2025-01-19 12:53:45,152 INFO    Thread-4  :13698 [wandb_run.py:finish():1347] finishing run maestria-julian/Ablation_LLMs/61fdt6yv
2025-01-19 12:53:45,153 INFO    Thread-4  :13698 [wandb_run.py:_atexit_cleanup():1663] got exitcode: 1
2025-01-19 12:53:45,153 INFO    Thread-4  :13698 [wandb_run.py:_restore():1635] restore
2025-01-19 12:53:45,160 INFO    Thread-4  :13698 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 762
  total_bytes: 762
}

